.pre-commit-config.yaml

```diff
@@ -58,7 +58,7 @@ repos:
   #     - id: rst-linter
   #       exclude: docs
 
-  - repo: https://gitlab.com/pycqa/flake8
+  - repo: https://github.com/pycqa/flake8
```

Comment:
this was erroring for me and asking me to sign into gitlab, it works with github?

Reply:
Not sure. Probably fine to use GitHub instead